src/h5web/visualizations/shared/AxisSystem.module.css

```diff
@@ -1,11 +1,13 @@
 .axisSystem {
-  position: relative;
-  top: 0;
```

Comment:
These weren't actually needed, since the `Html` component is already positioned absolutely.